examples/cross-browser.py

```diff
@@ -84,11 +85,12 @@ async def main():
     # Puppeteer:
     # await page.goto('https://news.ycombinator.com/');
     # https://github.com/puppeteer/puppeteer/blob/4c3caaa3f99f0c31333a749ec50f56180507a374/examples/cross-browser.js#L34
+    requestedPagePath = f'file://{Path(__file__).parent.resolve()}/app.html'
```

Comment:
```suggestion
    # To avoid the network dependency in this test, we use a local, static copy.
    requestedPagePath = f'file://{Path(__file__).parent.resolve()}/app.html'
```